crates/iceberg/src/avro/schema.rs

```diff
@@ -43,6 +43,41 @@ const MAP_LOGICAL_TYPE: &str = "map";
 // This const may better to maintain in avro-rs.
 const LOGICAL_TYPE: &str = "logicalType";
 
+fn literal_to_json(literal: &crate::spec::Literal) -> Result<Value> {
```

Comment:
whats the reason behind serializing to json?

Reply:
Used the other method, see below.